boards/arm/stm32f746g_disco/doc/stm32f746g_disco.rst

```diff
@@ -0,0 +1,199 @@
+.. _stm32f746g_disco_board:
+
+ST STM32F746G Discovery
+#######################
+
+Overview
+********
+
+The discovery kit enables a wide diversity of applications taking benefit
+from audio, multi-sensor support, graphics, security, security, video,
+and high-speed connectivity features. Important board features include:
+
+- STM32F746NGH6 microcontroller featuring 1 Mbytes of Flash memory and 340 Kbytes of RAM, in BGA216 package
+- On-board ST-LINK/V2-1 supporting USB re-enumeration capability
+- Five power supply options:
+
+    - ST LINK/V2-1
```

Comment:
The sublist should be indented to the first character of the text of the parent list:

    - Five power supply options:

      - ST LINK/V2-1

If you indent it too much, it changes the way the sublist is rendered.

Reply:
I have fixed it and updated the commit.